solidity/migrations/relay-config.json

```
@@ -0,0 +1,16 @@
+{
+  "forceOnDemandSPV": false,
+  "forceTestnetRelay": false,
+  "genesis": {
```

We already have `genesis` property under `bitcoinMain` and `bitcoinTest`.
```suggestion
  "init": {
```